composer.json

```diff
@@ -20,33 +20,33 @@
     "require": {
         "php": "^7.1",
         "imagine/imagine": "^0.7.1|^1.1",
-        "symfony/asset": "^3.4|^4.2|^5.0",
-        "symfony/filesystem": "^3.4|^4.2|^5.0",
-        "symfony/finder": "^3.4|^4.2|^5.0",
-        "symfony/framework-bundle": "^3.4|^4.2|^5.0",
-        "symfony/options-resolver": "^3.4|^4.2|^5.0",
-        "symfony/process": "^3.4|^4.2|^5.0",
-        "symfony/templating": "^3.4|^4.2|^5.0",
-        "symfony/translation": "^3.4|^4.2|^5.0"
+        "symfony/asset": "^3.4|^4.3|^5.0",
+        "symfony/filesystem": "^3.4|^4.3|^5.0",
+        "symfony/finder": "^3.4|^4.3|^5.0",
+        "symfony/framework-bundle": "^3.4|^4.3|^5.0",
+        "symfony/options-resolver": "^3.4|^4.3|^5.0",
+        "symfony/process": "^3.4|^4.3|^5.0",
+        "symfony/templating": "^3.4|^4.3|^5.0",
+        "symfony/translation": "^3.4|^4.3|^5.0"
     },
     "require-dev": {
         "ext-gd": "*",
         "amazonwebservices/aws-sdk-for-php": "^1.0",
         "aws/aws-sdk-php": "^2.4",
         "doctrine/cache": "^1.1",
         "doctrine/orm": "^2.3",
-        "enqueue/enqueue-bundle": "^0.9",
+        "enqueue/enqueue-bundle": "^0.9|^0.10",
```

Comment:
0.9 was kept because 0.10 does not support 3.4. So `0.9|0.10` allows Symfony 3.4 through `^5.0`